templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.Mongo/server/mongo/mongo_service.py

```diff
@@ -0,0 +1,40 @@
+from flask import jsonify, make_response, request
+from .mongo_client import *
```

Comment:
Talked to @njindal239 about `import *` since it is usually discouraged. @njindal239 will address it in another PR.